src/types/purchaseOrder.ts

```diff
@@ -0,0 +1,10 @@
+interface PurchaseOrder {
```

Comment:
When naming an interface, you should add a prefix "I" followed by the name of the interface.